usr/share/rear/output/PXE/default/810_create_pxelinux_cfg.sh

```diff
@@ -32,19 +32,50 @@ if [[ ! -z "$PXE_CONFIG_URL" ]] ; then
     fi
     chmod 444 "$PXE_LOCAL_PATH/$PXE_CONFIG_FILE"
 else
-    # legacy way using PXE_LOCAL_PATH default
-    cat >"$PXE_LOCAL_PATH/$PXE_CONFIG_FILE" <<EOF
-    $(test -s $(get_template "PXE_pxelinux.cfg") && cat $(get_template "PXE_pxelinux.cfg"))
-    display $OUTPUT_PREFIX_PXE/$PXE_MESSAGE
-    say ----------------------------------------------------------
-    say rear = disaster recover this system with Relax-and-Recover
-    label rear
-	kernel $OUTPUT_PREFIX_PXE/$PXE_KERNEL
-	append initrd=$OUTPUT_PREFIX_PXE/$PXE_INITRD root=/dev/ram0 vga=normal rw $KERNEL_CMDLINE $PXE_RECOVER_MODE
+    # UEFI case, cf. https://github.com/rear/rear/issues/2359
+    EFIMT="$( mount -v | grep /sys/firmware/efi/efivars | wc -l )"
+    EFIMGR="$( efibootmgr -v | grep "BootOrder" | wc -l )"
```

Comment:
I wonder if there exists a "BootOrder" entry in any case when UEFI is used?
I think the UEFI firmware can boot a default/fallback EFI binary
basically without any UEFI configuration entries.